VerifyBamID/2.0.1--h32f71e1_0/CheckContamination.nf

```diff
@@ -0,0 +1,64 @@
+process CheckContamination {
+    tag {"VerifyBamID CheckContamination ${sample_id}.${int_tag}"}
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0'
+    label 'VERIFYBAMID_2_0_1_h32f71e1_0_CheckContamination'
+    container = 'quay.io/biocontainers/verifybamid2:2.0.1--h32f71e1_0'
+    shell = ['/bin/bash', '-eo', 'pipefail']
+
+    input:
+        tuple (
+            sample_id, 
+            path(bam), 
+            path(bai))
+
+    output:
+        tuple (
+            sample_id, 
+            path("${output_prefix}.selfSM"), 
+            stdout)
+
+    script:
+        output_prefix = "${sample_id}.${params.library_strategy}.contamination"
+
+        """
+        # creates a ${output_prefix}.selfSM file, a TSV file with 2 rows, 19 columns.
+        # First row are the keys (e.g., SEQ_SM, RG, FREEMIX), second row are the associated values
+        verifybamid2 \
+        --Reference ${params.genome} \
+        --BamFile ${bam} \
+        --SVDPrefix ${params.contamination_path_prefix} \
+        --UDPath ${params.contamination_sites_ud} \
+        --MeanPath ${params.contamination_sites_mu} \
+        --BedPath ${params.contamination_sites_bed} \
+        --Verbose \
+        --NumPC 4 \
+        --Output  ${output_prefix} \
+        1>/dev/null
```

Comment:
What does the tool output to stdout? I think we should not redirect to /dev/null and store the output in .command.out (automatically done by nextflow). Might be useful output when the tool crashes?

Reply:
Its a correction on contamination calculated in the python part.
In this case the output appeared in .command.out as well, maybe /dev/null is also the default .command.out?

- [ ] Remove /dev/null

Reply:
- [ ] add ancestry file to output